src/mercury_engine_data_structures/formats/btunda.py

```diff
@@ -0,0 +1,41 @@
+
+import construct
+from construct import Container
+
+from mercury_engine_data_structures.formats import BaseResource, standard_format
+from mercury_engine_data_structures.game_check import Game
+
+VALID_BTUNDA_VERSIONS = [
+    0x02000077, # 1.0.0
+    0x02000080, # 2.1.0
+]
+
+class Btunda(BaseResource):
+    """
+    /!\\ /!\\ /!\\ READ THIS WHEN USING!!! /!\\ /!\\ /!\\
+
+    This format has TWO VERSIONS between 1.0.0 and 2.1.0!
+
+    To prevent unexpected behavior across versions, do not use:\n
+    - any key containing EasyMode in Player|CTunablePlayerLifeComponent\n
+    - anything in CTunableBossRushManager
+    """
+
+    @classmethod
+    def construct_class(cls, target_game: Game, version:int) -> construct.Construct:
+        return standard_format.create('base::tunable::CTunableManager', version)
```

Comment:
I'd actually leave this method unimplemented, and instead implement a custom `build()` method just like the custom `parse()`.